runtime/lua/vim/treesitter/query.lua

```diff
@@ -480,6 +479,32 @@ local directive_handlers = {
     end
     metadata[id].text = text:gsub(pred[3], pred[4])
   end,
+  -- Excludes child languages in injections
```

Comment:
If anybody has a better description, I'd be happy to use that...

Reply:
I'm not really sure what this does? @theHamsta do you know?

Reply:
This is the default behavior of tree-sitter (and Helix, and presumably others) when handling injections. I really think we should make this the default behavior as well (because this is one of the blockers for sharing queries with others).